ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -32,21 +33,38 @@ def enable_appsec_rc():
     from ddtrace import tracer
 
     if _appsec_rc_features_is_enabled():
+        from ddtrace.appsec._constants import PRODUCTS
         from ddtrace.internal.remoteconfig import RemoteConfig
-        from ddtrace.internal.remoteconfig.constants import ASM_DATA_PRODUCT
-        from ddtrace.internal.remoteconfig.constants import ASM_FEATURES_PRODUCT
 
-        RemoteConfig.register(ASM_FEATURES_PRODUCT, appsec_rc_reload_features(tracer))
-        RemoteConfig.register(ASM_DATA_PRODUCT, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(PRODUCTS.ASM_FEATURES, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(PRODUCTS.ASM_DATA, appsec_rc_reload_features(tracer))  # IP Blocking
+        RemoteConfig.register(PRODUCTS.ASM, appsec_rc_reload_features(tracer))  # Exclusion Filters & Custom Rules
+        RemoteConfig.register(PRODUCTS.ASM_DD, appsec_rc_reload_features(tracer))  # DD Rules
+
+
+def _loading_rules(features, feature, message, rule_list):
+    # type: (Mapping[str, Any], str, str, list[Any]) -> None
+    rules = features.get(feature, [])
+    if rules:
+        try:
+            rule_list += json.loads(rules)
+            log.debug("Reloading Appsec %s: %s", message, rules)
+        except json.decoder.JSONDecodeError:
```

Comment:
I think this fails for Python2, you have to do the `JSONDecodeError = ValueError` trick

Reply:
thank you, I fixed it.